### ISSUE TITLE
Failed install on Ubuntu 14.04.3 LTS

### ISSUE THREAD

**Issue by tswr** [Valloric/ycmd#508]
`subprocess.CalledProcessError: Command '['/usr/bin/python', '/home/tswr/vim-tswr/.vim/bundle/YouCompleteMe/third_party/ycmd/build.py', '--clang-completer']' returned non-zero exit status 1`

This seems to fix the problem (in ~/.vim/bundle/YouCompleteMe/third_party/ycmd):
```
diff --git a/build.py b/build.py
index 5ab86d2..1773bd4 100755
--- a/build.py
+++ b/build.py
@@ -170,7 +170,7 @@ def FindPythonLibrariesOnLinux():
   ldconfig_output = CheckOutput( [ 'ldconfig', '-p' ] ).strip().decode( 'utf8' )
   for line in ldconfig_output.splitlines():
     match = LIBRARY_LDCONFIG_REGEX.search( line )
-    if match and match.group( 'library' ) == python_library:
+    if match and match.group( 'library' ).startswith(python_library):
       return match.group( 'path' ), python_include

   sys.exit( NO_PYTHON_LIBRARY_ERROR )
```

**Comment by micbou** [comment]
Could you try PR #499? It should also fix your issue.

**Comment by micbou** [comment]
No answer. I consider this is fixed by PR #499.

**Comment by tswr** [comment]
Sorry, I missed your message. Yes, (\.\d+)* part in regexp should fix this.
